config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml

```diff
@@ -195,6 +195,51 @@ spec:
           - 172.17.0.86
           ipAddressPool: internalapi
           sharedIP: false
+  heat:
+    enabled: false
+    template:
+      customServiceConfig: "# add your customization here"
+      databaseInstance: openstack
+      databaseUser: "heat"
+      rabbitMqClusterName: rabbitmq
+      debug:
+        bootstrap: false
+        dbSync:
```

Comment:
is this intentionally not set?

Reply:
anyway i think it would be good to only keep required parameters here instead of all like it's done for other operators.

Reply:
I set the value. But, in general I agree we should only keep required params here. Although, we should probably do it consistently for everything. Maybe as a seperate piece of work, we can look at reducing the size of these samples?